repo/packages/M/minio/6/config.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "properties": {
+        "service": {
+            "type": "object",
+            "description": "Service properties for Minio on DC/OS",
+            "properties": {
+                "name": {
+                    "description": "Name of this service instance",
+                    "type": "string"
+                },
+                "cpus": {
+                    "description": "CPU shares to allocate to service instance",
+                    "type": "number",
+                    "default": 1,
+                    "minimum": 0.5
+                },
+                "mem": {
+                    "description": "Memory to allocate to service instance",
+                    "type": "number",
+                    "default": 2048,
+                    "minimum": 2048
+                }
+            },
+            "required": [
+                "name",
+                "cpus",
+                "mem"
+            ]
+        },
+        "storage": {
+            "type": "object",
+            "description": "Storage properties for Minio on DC/OS",
+            "properties": {
+                "data-size": {
+                    "type": "number",
+                    "description": "Size of data volume in MiB"
+                }
+            },
+            "required": [
+                "data-size"
+            ]
+
+        },
+        "credentials": {
+            "type": "object",
+            "description": "Credentials to authenticate with Minio server",
+            "properties": {
+                "access-key": {
+                    "type": "string",
+                    "description": "Access Key",
+                    "default": "minio"
+                },
+                "secret-key": {
+                    "type": "string",
+                    "description": "Secret Key",
+                    "default": "minio123"
+                }
+            }
+        },
+        "networking": {
+            "description": "Networking properties for Minio on DC/OS",
+            "type": "object",
+            "properties": {
+                "service-port": {
+                    "description": "Minio service port",
+                    "type": "number"
```

Comment:
@ryadav88 `service-port` configuration option having a default is problematic when 2 or more instances of minio are installed with the same (default) port. IIUC Marathon/Marathon-LB don't reject instances using the same service port. Now, we are left to choose between one-click default installation and incorrect service deployment. I am inclined towards preventing incorrect service deployment by _not_ providing a default for `service-port`. If you think it is uncommon for users to make this mistake, then I shall add a default. Let me know your thoughts on this.